project/Dependencies.scala

```diff
@@ -16,9 +16,19 @@
 import sbt._
 
 object Dependencies {
-  val baseDependencies = List(
-    "org.apache.spark" %% "spark-core" % "2.4.4" % Provided,
-    "org.apache.spark" %% "spark-sql"  % "2.4.4" % Provided,
-    "org.scalatest"    %% "scalatest" % "3.0.5"  % Test
+  def sparkVersion: String = sys.props.getOrElse("SPARK_VERSION", "2.4.7")
+
+  private val scalatestVersion = "3.0.5"
+
+  def getScalaDependency(scalaVersion: String): ModuleID = "org.scala-lang" % "scala-library" % scalaVersion % Provided
```

Comment:
Where is this used?

Reply:
Removed. There was a plan to make the library even lighter, but I misunderstood the scala dependency in it. So I, in the end, removed this line